doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Sonata project contribution
+
+Thanks for you interest onto Sonata projects!
+
+## Summary
+
+* [Issues]()
+* [Pull Requests](#pull-requests)
+* [Label rules]()
+
+## Pull Requests
+
+All the sonata team will be glad to review your code changes propositions! :smile:
+
+But please, read the following before.
+
+### Coding style
+
+Each project follows [PSR-1](http://www.php-fig.org/psr/psr-1/), [PSR-2](http://www.php-fig.org/psr/psr-2/)
+and [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html) for coding style,
+[PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading.
+
+Please [install PHP Coding Standard Fixer](http://cs.sensiolabs.org/#installation)
+and run this command before committing your modifications:
+
+```bash
+php-cs-fixer fix --verbose
+```
+
+### Writing a Pull Requests
```

Comment:
> I remember one commit (from you?) today doing exactly the opposite, it was a one-liner :

Do what I say not what I do. :stuck_out_tongue: 

More seriously. Indeed, but this kind of case should be avoid **as possible**.

> You want it to work.
> You can't have two parrallel branches because possible conflicts and you need your app to work while waiting for the merge.

Just create another branch just for you one your fork merging the both cases.

Anyway, like @greg0ire says, installed tool preventing bad coding style and other will reduce this cases, which should/must be rares IMO.